// graphql-scalars/dist/commonjs/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const DateTime_1 = require("./DateTime");
exports.DateTime = DateTime_1.default;
const NonPositiveInt_1 = require("./NonPositiveInt");
exports.NonPositiveInt = NonPositiveInt_1.default;
const PositiveInt_1 = require("./PositiveInt");
exports.PositiveInt = PositiveInt_1.default;
const NonNegativeInt_1 = require("./NonNegativeInt");
exports.NonNegativeInt = NonNegativeInt_1.default;
exports.UnsignedInt = NonNegativeInt_1.default;
const NegativeInt_1 = require("./NegativeInt");
exports.NegativeInt = NegativeInt_1.default;
const NonPositiveFloat_1 = require("./NonPositiveFloat");
exports.NonPositiveFloat = NonPositiveFloat_1.default;
const PositiveFloat_1 = require("./PositiveFloat");
exports.PositiveFloat = PositiveFloat_1.default;
const NonNegativeFloat_1 = require("./NonNegativeFloat");
exports.NonNegativeFloat = NonNegativeFloat_1.default;
exports.UnsignedFloat = NonNegativeFloat_1.default;
const NegativeFloat_1 = require("./NegativeFloat");
exports.NegativeFloat = NegativeFloat_1.default;
const EmailAddress_1 = require("./EmailAddress");
exports.EmailAddress = EmailAddress_1.default;
const URL_1 = require("./URL");
exports.URL = URL_1.default;
const PhoneNumber_1 = require("./PhoneNumber");
exports.PhoneNumber = PhoneNumber_1.default;
const PostalCode_1 = require("./PostalCode");
exports.PostalCode = PostalCode_1.default;
const RegularExpression_1 = require("./RegularExpression");
exports.RegularExpression = RegularExpression_1.default;
const BigInt_1 = require("./BigInt");
exports.BigInt = BigInt_1.default;
exports.Long = BigInt_1.default;
const scalars_1 = require("./scalars");
exports.DateTimeScalar = scalars_1.DateTimeScalar;
exports.EmailAddressScalar = scalars_1.EmailAddressScalar;
exports.NegativeFloatScalar = scalars_1.NegativeFloatScalar;
exports.NegativeIntScalar = scalars_1.NegativeIntScalar;
exports.NonNegativeFloatScalar = scalars_1.NonNegativeFloatScalar;
exports.NonNegativeIntScalar = scalars_1.NonNegativeIntScalar;
exports.NonPositiveFloatScalar = scalars_1.NonPositiveFloatScalar;
exports.NonPositiveIntScalar = scalars_1.NonPositiveIntScalar;
exports.PhoneNumberScalar = scalars_1.PhoneNumberScalar;
exports.PositiveFloatScalar = scalars_1.PositiveFloatScalar;
exports.PositiveIntScalar = scalars_1.PositiveIntScalar;
exports.PostalCodeScalar = scalars_1.PostalCodeScalar;
exports.RegularExpressionScalar = scalars_1.RegularExpressionScalar;
exports.UnsignedFloatScalar = scalars_1.UnsignedFloatScalar;
exports.UnsignedIntScalar = scalars_1.UnsignedIntScalar;
exports.URLScalar = scalars_1.URLScalar;
exports.BigIntScalar = scalars_1.BigIntScalar;
exports.LongScalar = scalars_1.LongScalar;
var scalars_2 = require("./scalars");
exports.OKGScalarDefinitions = scalars_2.default;
exports.default = {
    // as part of a default object
    DateTime: DateTime_1.default,
    PositiveInt: PositiveInt_1.default,
    NonNegativeInt: NonNegativeInt_1.default,
    UnsignedInt: NonNegativeInt_1.default,
    NegativeInt: NegativeInt_1.default,
    PositiveFloat: PositiveFloat_1.default,
    NonNegativeFloat: NonNegativeFloat_1.default,
    UnsignedFloat: NonNegativeFloat_1.default,
    NegativeFloat: NegativeFloat_1.default,
    EmailAddress: EmailAddress_1.default,
    URL: URL_1.default,
    PhoneNumber: PhoneNumber_1.default,
    PostalCode: PostalCode_1.default,
    RegularExpression: RegularExpression_1.default,
    BigInt: BigInt_1.default,
    Long: BigInt_1.default,
};
