// graphql-scalars/dist/commonjs/ScalarsModule.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const core_1 = require("@graphql-modules/core");
const scalars_1 = require("./scalars");
const DateTime_1 = require("./DateTime");
const NonPositiveInt_1 = require("./NonPositiveInt");
const PositiveInt_1 = require("./PositiveInt");
const NonNegativeInt_1 = require("./NonNegativeInt");
const NegativeInt_1 = require("./NegativeInt");
const NonPositiveFloat_1 = require("./NonPositiveFloat");
const PositiveFloat_1 = require("./PositiveFloat");
const NonNegativeFloat_1 = require("./NonNegativeFloat");
const NegativeFloat_1 = require("./NegativeFloat");
const EmailAddress_1 = require("./EmailAddress");
const URL_1 = require("./URL");
const PhoneNumber_1 = require("./PhoneNumber");
const PostalCode_1 = require("./PostalCode");
const BigInt_1 = require("./BigInt");
exports.ScalarsModule = new core_1.GraphQLModule({
    typeDefs: scalars_1.default,
    resolvers: {
        DateTime: DateTime_1.default,
        PositiveInt: PositiveInt_1.default,
        NonPositiveInt: NonPositiveInt_1.default,
        NonNegativeInt: NonNegativeInt_1.default,
        UnsignedInt: NonNegativeInt_1.default,
        NegativeInt: NegativeInt_1.default,
        PositiveFloat: PositiveFloat_1.default,
        NonPositiveFloat: NonPositiveFloat_1.default,
        NonNegativeFloat: NonNegativeFloat_1.default,
        UnsignedFloat: NonNegativeFloat_1.default,
        NegativeFloat: NegativeFloat_1.default,
        EmailAddress: EmailAddress_1.default,
        URL: URL_1.default,
        PhoneNumber: PhoneNumber_1.default,
        PostalCode: PostalCode_1.default,
        BigInt: BigInt_1.default,
        Long: BigInt_1.default,
    }
});
