// graphql-scalars/dist/commonjs/scalars.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.DateTimeScalar = 'scalar DateTime';
exports.EmailAddressScalar = 'scalar EmailAddress';
exports.NegativeFloatScalar = 'scalar NegativeFloat';
exports.NegativeIntScalar = 'scalar NegativeInt';
exports.NonNegativeFloatScalar = 'scalar NonNegativeFloat';
exports.NonNegativeIntScalar = 'scalar NonNegativeInt';
exports.NonPositiveFloatScalar = 'scalar NonPositiveFloat';
exports.NonPositiveIntScalar = 'scalar NonPositiveInt';
exports.PhoneNumberScalar = 'scalar PhoneNumber';
exports.PositiveFloatScalar = 'scalar PositiveFloat';
exports.PositiveIntScalar = 'scalar PositiveInt';
exports.PostalCodeScalar = 'scalar PostalCode';
exports.RegularExpressionScalar = 'scalar RegularExpression';
exports.UnsignedFloatScalar = 'scalar UnsignedFloat';
exports.UnsignedIntScalar = 'scalar UnsignedInt';
exports.URLScalar = 'scalar URL';
exports.BigIntScalar = 'scalar BigInt';
exports.LongScalar = 'scalar Long';
exports.default = [
    exports.DateTimeScalar,
    exports.EmailAddressScalar,
    exports.NegativeFloatScalar,
    exports.NegativeIntScalar,
    exports.NonNegativeFloatScalar,
    exports.NonNegativeIntScalar,
    exports.NonPositiveFloatScalar,
    exports.NonPositiveIntScalar,
    exports.PhoneNumberScalar,
    exports.PositiveFloatScalar,
    exports.PositiveIntScalar,
    exports.PostalCodeScalar,
    exports.RegularExpressionScalar,
    exports.UnsignedFloatScalar,
    exports.UnsignedIntScalar,
    exports.URLScalar,
    exports.BigIntScalar,
    exports.LongScalar,
];
